services/apps/src/apps/routes/schemas/groups.clj

```diff
@@ -58,15 +58,6 @@
 (s/defschema GroupMembersUpdate
   {:members (describe [NonBlankString] "The new list of member subject IDs.")})
 
-(s/defschema GroupMemberSubjectUpdateResponse
-  {:success
-   (describe Boolean "True if the user was added successfully")
-
-   :subject_id
-   (describe NonBlankString "The subject ID.")
-
-   (s/optional-key :subject_name)
-   (describe NonBlankString "The subject name.")})
-
 (s/defschema GroupMembersUpdateResponse
-  {:results (describe [GroupMemberSubjectUpdateResponse] "The list of membership update results.")})
```

Comment:
I changed the response body of the `PUT /admin/groups/workshop/members` endpoint in order to prevent the UI from having to chain service calls to refresh the member list.